test/zypper_test/test_serviceplugin.py

```diff
@@ -6,10 +6,10 @@
 import subprocess
 import tempfile
 
-from nose.plugins.attrib import attr
+from test import subman_marker_zypper
 
 
-@attr('zypper')
+@subman_marker_zypper
 class TestServicePlugin(TestCase):
```

Comment:
this also needs a "functional" marker, since this is a functional test (it runs sub-man as installed, including the zypper plugin); can be an own commit